README.md

```diff
@@ -31,7 +31,7 @@ Include the package using `require()` or ES2015/TypeScript `import` statement:
 const KeyCode = require('keycode-js');
 
 // ES2015+
-import KeyCode from 'keycode-js';
+import * as KeyCode from 'keycode-js';
```

Comment:
@mesaugat Will the old syntax still work? I think it still should or it will be a breaking change.

Reply:
I am not sure it will work because we don't have a default export. Will need to check.

Reply:
It should work.
https://github.com/kabirbaidhya/keycode-js/blob/master/dist/keycode.cjs.js#L3

Reply:
Oops it doesn't work. I tried it in a React App. I am doubtful, did it work before this?

Reply:
It is a breaking change if we need to support default export. Ok, let's come back to this in another thread.

Reply:
Do we need to have a default export? I think we can release this as `2.0.0`

Reply:
Yes. It's for 2.0.0